### PR TITLE
chore(mitre gcp): add mitre mapping for gcp

### PR DESCRIPTION
**Pull request by n4ch04**
### Context

We need to cover mitre attack mapping for gcp

### Description

Include mitre attack mapping for gcp

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/lib/outputs/file_descriptors.py [hunk 1]
```diff
@@ -170,7 +188,7 @@ def fill_file_descriptors(output_modes, output_directory, output_filename, provi
                             file_descriptor = initialize_file_descriptor(
                                 filename,
                                 output_mode,
-                                Check_Output_MITRE_ATTACK,
+                                MitreAttackAWS,
```
Comment (pedrooot) [inline prowler/lib/outputs/file_descriptors.py]: `elif provider.type == "azure":` appears two times in this method, could you please unify them?

**Review by pedrooot — CHANGES_REQUESTED** [review]
Great work! @n4ch04 Please review my comments 🙏🏼

File: prowler/compliance/azure/mitre_attack_azure.json [hunk 2]
```diff
@@ -0,0 +1,2496 @@
...
+      ],
+      "Description": "Adversaries may attempt to get a listing of services running on remote hosts and local network infrastructure devices, including those that may be vulnerable to remote software exploitation. Common methods to acquire this information include port and/or vulnerability scans using tools that are brought onto a system.",
+      "TechniqueURL": "https://attack.mitre.org/techniques/T1046/",
+      "Checks": [
+        "network_http_internet_access_restricted",
+        "network_rdp_internet_access_restricted",
+        "network_ssh_internet_access_restricted",
+        "network_udp_internet_access_restricted",
+        "network_watcher_enabled",
+        "networkfirewall_in_all_vpc"
```
Comment (pedrooot) [inline prowler/compliance/azure/mitre_attack_azure.json]:
```suggestion
```
"networkfirewall_in_all_vpc" it's not a check for azure

Comment (n4ch04) [reply prowler/compliance/azure/mitre_attack_azure.json]: Deleted

File: prowler/lib/outputs/file_descriptors.py (hunk 1 above)
Comment (n4ch04) [reply prowler/lib/outputs/file_descriptors.py]: Solved

**Review by pedrooot — DISMISSED** [review]
🔝  work @n4ch04

File: prowler/compliance/gcp/cis_2.0_gcp.json [hunk 3]
```diff
@@ -396,9 +396,7 @@
     {
       "Id": "2.13",
       "Description": "GCP Cloud Asset Inventory is services that provides a historical view of GCP resources and IAM policies through a time-series database. The information recorded includes metadata on Google Cloud resources, metadata on policies set on Google Cloud projects or resources, and runtime information gathered within a Google Cloud resource.",
-      "Checks": [
-        "serviceusage_cloudasset_inventory_enabled"
-      ],
+      "Checks": [],
```
Comment (MrCloudSec) [inline prowler/compliance/gcp/cis_2.0_gcp.json]:
```suggestion
      "Checks": [
        "iam_cloud_asset_inventory_enabled"
      ],
```

**Review by MrCloudSec — DISMISSED** [review]

**Review by MrCloudSec — APPROVED** [review]

**Pull request merged**